src/modules/search.js

```diff
@@ -140,11 +144,15 @@ class Search {
             });
           }
 
+          this.eventDispatcher.queue(this.module, method, 'completed', json);
```

Comment:
Might be a little too much indirection here with the variables.  It might be easier to read without them, like 
```javascript
this.eventDispatcher.queue('search', 'getSearchResults`, 'completed', json)
```

Reply:
Agreed - a bit overabstracted.